### ISSUE TITLE
[simprod-scripts] update error message in PropagatePhotons (Trac #1903)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1895]
Misleading error message in `segments.HybridPhotonicsCLSim.PropagatePhotons:145`: hybrid mode supports SpiceLea but error message was not updated accordingly.

Migrated from https://code.icecube.wisc.edu/ticket/1903
```json
{
    "status": "closed", 
    "changetime": "2016-11-03T21:21:24", 
    "description": "Misleading error message in `segments.HybridPhotonicsCLSim.PropagatePhotons:145`: hybrid mode supports SpiceLea but error message was not updated accordingly.", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1478208084587955", 
    "component": "combo simulation", 
    "summary": "[simprod-scripts] update error message in PropagatePhotons", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-10-27T13:48:08", 
    "milestone": "", 
    "owner": "juancarlos", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161102T16:57:48: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "" to "juancarlos"

**Comment by icecube-trac** [comment]
Trac update at 20161103T21:21:24: 
* juancarlos commented:

> In changeset 151319:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="151319"
> Removed SpiceLea from tuple of supported icemodes in Hybrid mode and made
> error message clearer. Closes #1895
> ```


* juancarlos changed status from "assigned" to "closed"
* juancarlos changed resolution from "" to "fixed"
